### ISSUE TITLE
Black Screen After Opening

### ISSUE THREAD

**Issue by Manik3112** [mozillahispano/openwapp#150]
I Downloaded Openwapp Yesterday in my phone Intex Cloud Fx and found not working properly..
When i open it first time it asked to read memory card, after i confirmed the Black Screen Occurs untill i close it.

**Comment by bobcanthelpyou** [comment]
Which Firefox OS version does run on your Intex Cloud Fx phone? A restart of your phone does not change that behavior?

**Comment by Timmaks** [comment]
I think  you found a (l10n) issue with the confirmation screen. Once you have granted the permissions to openwapp you will not see this issue anymore. 

Witch version of openwapp do you use, fxos and language?

**Comment by Manik3112** [comment]
FFOs is 1.3 nd Even Granting All the Permission it still doesn't work..
Its Seems the Registration Windows isn't Working...

**Comment by bobcanthelpyou** [comment]
Could you try to connect your phone with the WebIDE of Firefox and post the console output here?

**Comment by Manik3112** [comment]
i thought the new version have a problem.. coz i'm using it since last two months..
but i reset my phone.. nd installed it again.. it sucks

**Comment by bobcanthelpyou** [comment]
What language do you use on your phone? It is not possible for you to connect your phone with WebIDE of Firefox? https://developer.mozilla.org/en/docs/Tools/WebIDE

* did you restart your phone?
* could you try to setup a different language of Firefox OS?

At the moment i do not run into that issue with Firefox OS Simulator v1.3 and OpenWapp v1.8.1-0-gb77614d. And there is no permission screen for me.

**Comment by nagaprakashv** [comment]
Me too facing the issue.
There is no problem for me with previous version of Openwapp. Now only updated to current version and provided all of permissions. But, the registration page have some bug i think.
Using Spice Mi-fx1 running on a Indian version of fxos (1.3) and the language in setting is only English.
Help me...

**Comment by nagaprakashv** [comment]
I'm sure it's not a handset problem...

**Comment by bobcanthelpyou** [comment]
@nagaprakashv and @Manik3112 could you please try to connect your phone to Firefox WebIDE and check the console output of Openwapp there? Maybe there are some useful error messages or something else about that issue.

**Comment by tripad** [comment]
@bobcanthelpyou Having the same issue

This my console output
```
"[l10n] app://80704eed-18e3-41ef-88bf-2bfa3a35559d/locales/../components/fxosRate/locales/fxosrate.en-US.properties not found." 
    at consoleWarn (app://80704eed-18e3-41ef-88bf-2bfa3a35559d/scripts/l10n.js:79:7) l10n.js:79:7
```

Device : ZTE Open C
B2G 2.5.0.0-Prerelease Nightly Build

Tried changing locale to EN-GB still doesnt go past the blank screen

**Comment by Timmaks** [comment]
thanks i think i know the problem now, will fix it soon.

**Comment by Timmaks** [comment]
this should be fixed with commit 21083c9bc2bd311c7a4e6798ea4c32ea4440dd35

**Comment by Timmaks** [comment]
please reopen if it still exist in the next version on marketplace
